packages/grid/_modules_/grid/components/panel/PanelContent.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Did you consider using the `styled()` helper instead? It could be simpler and help with v5 migration.

x3 (the other Panel components)

Reply:
That's a separate issue that can be done in the whole grid

Reply:
Ok, I'm asking because at's this point, the probability we deprecate makeStyles/withStyles in v5 is over 99%. I was wondering about the potential to save time in a future migration to introduce new more "compatible" code.

Reply:
Let's do a separate issue, so we don't have a mix in the codebase of makestyle and styled.

Reply:
Ok

For the curious ones, `styled` is a "thin" wrapper on top of makeStyles: https://github.com/mui-org/material-ui/blob/7bc6e3e10ada78a6133292979abca5ee5b9d1779/packages/material-ui-styles/src/styled/styled.js#L6.